RecoMET/METFilters/python/hcalLaserEventFilter_cfi.py

```diff
@@ -1,27 +1,14 @@
-import FWCore.ParameterSet.Config as cms
+from RecoMET.METFilters.hcallaserevent_cfi import * # from fillDescriptions
 
 try:
     from hcalLaserBadEvents_2011 import badEvents
 except:
     print "<hcalLaserEventFilter_cfi> WARNING!  No explicit list 'badEvents.py' containing bad HCAL laser run/events was found!"
     badEvents=[]
 
+#hcalLaserEventFilter = cms.EDFilter("HcalLaserEventFilter",)
 hcalLaserEventFilter = cms.EDFilter("HcalLaserEventFilter",

```

Comment:
Now that you have imported hcallaserevent (as exiting from the filldescriptions, with the default parameters set) you should better clone hcalLaserEventFilter out of it, and only modify the parameter (BadRunEventNumbers) that needs to